code/modules/clothing/suits/armor.dm

```diff
@@ -634,6 +634,7 @@
 	)
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 	style_coverage = COVERS_TORSO|COVERS_UPPER_ARMS|COVERS_UPPER_LEGS
+	accompanying_object = /obj/item/clothing/head/armor/faceshield/paramedic
```

Comment:
if you're going to do that, you should probably make sure this doesn't make there be two helmets everywhere this is.